src/test/java/seedu/address/logic/commands/GroupCommandTest.java

```diff
@@ -0,0 +1,20 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+
+public class GroupCommandTest {
+    private Model model = new ModelManager();
+
+    @Test
+    public void testNonRecordedName() throws CommandException {
```

Comment:
Please follow the naming convention for test methods as follow:
![image](https://user-images.githubusercontent.com/70432640/140618255-659aac30-5bed-4a1a-974e-baeae13928cd.png)